api/karlchen.proto

```diff
@@ -264,6 +265,12 @@ enum Rank {
 }
 
 
+enum TrickEvent {
+    FOXCAUGHT = 0;
```

Comment:
I'd prefer sepating words by underscores (in line with the [style guide](https://developers.google.com/protocol-buffers/docs/style#enums)):
```
FOX_CAUGHT
DOPPELKOPF_SCORED
...